src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LambdaHandler.java

```diff
@@ -92,10 +91,13 @@ protected IndentLevel getIndentImpl() {
     public void checkIndentation() {
         // If the argument list is the first element on the line
         final DetailAST firstChild = getMainAst().getFirstChild();
```

Comment:
`getMainAst()` is called three times inside the same function. We can extract out in a final variable.

Reply:
Thanks, I applied this to both functions where I made changes. Done.

Reply:
resolved